GoogleForCreators/web-stories-wp
# Fix focus colour in the design and library panels

## Summary
The focus outline assumes that the colour should be set based on the background of the element. This PR aims to do that for most of the focusable elements in the library and design panel.


## Relevant Technical Choices



## To-do



## User-facing changes
|Before|After|
|----|----|
| <img width="448" alt="Screenshot 2021-05-07 at 14 58 47" src="https://user-images.githubusercontent.com/3294597/117446437-5d1cd000-af3c-11eb-82ec-595dee64cb8d.png"> | <img width="439" alt="Screenshot 2021-05-07 at 14 57 43" src="https://user-images.githubusercontent.com/3294597/117446361-45454c00-af3c-11eb-9923-294748e38af0.png"> |



## Testing Instructions
- Go through the design panel / library and verify that the inputs' focus shadow inner area looks the same colour as the background (see the difference between Before & After on the screenshots above).


### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #6858
